test-data/unit/check-modules.test

```diff
@@ -1640,3 +1640,15 @@ m = n  # E: Cannot assign multiple modules to name 'm' without explicit 'types.M
 [file n.py]
 
 [builtins fixtures/module.pyi]
+
+[case testPathAttributePresence]
```

Comment:
I think it would be good to add a Python 2 test as well. Those are created by giving them a name ending in `_python2`. (Several examples exist in this file.)